content/07.pathogenesis.md

```diff
@@ -124,42 +124,42 @@ The mechanism behind these insults has been suggested to be related to inflammat
+Hyperactivity of the pro-inflammatory response due to lung infection is commonly associated with acute lung injury and more rarely with the more severe manifestation, ARDS [@doi:10.1128/MMBR.05015-11].
+The heightened inflammatory response in the lungs can also serve as a source for systemic inflammation, or sepsis, which can lead to multi-organ failure [@doi:10.1128/MMBR.05015-11].
+The shift from local to systemic inflammation is a phenomenon often referred to broadly as a cytokine storm [@doi:10.1128/MMBR.05015-11] or a more precisely as cytokine release syndrome (CRS) [@doi:10.1186/s40425-018-0343-9].
+Sepsis is a known possible complication of pneumonia, and in an analysis of over 1400 US pneumonia patients, interleukin-6 (IL-6), tumor necrosis factor (TNF), and IL-1&beta; were found to be elevated at intake in patients who developed severe sepsis and/or ultimately deceased [@doi:10.1001/archinte.167.15.1655].
+IL-6 and TNF are pro-inflammatory cytokines, while IL-1&beta; is anti-inflammatory [@doi:10.1001/archinte.167.15.1655].
+However, this study reported that unbalanced pro-/anti-inflammatory cytokine profiles were rare, although they measured only the three cytokines listed above.
+Prior work therefore makes it clear that pulmonary infection and injury are associated with systemic inflammation through sepsis.
+While IL-6 is a biomarker sometimes used to assess cytokine storm activity in sepsis [@doi:10.1128/MMBR.05015-11], the relationship between cytokine profiles and the risks associated with sepsis may be more complex. 
+IL-6 is a pleiotropic cytokine that plays a role in both the inflammatory and anti-inflammatory responses and is associated with both healthy and pathological responses to viral threat [@doi:10.3389/fmicb.2019.01057].
+In this way, interleukins like IL-6 are integral to both the pro- and anti-inflammatory responses.
+
+The inflammatory response was identified early on as a potential driver of COVID-19 outcomes due to existing research in SARS and emerging research in COVID-19.
+In addition to the known role of cytokines in ARDS and lung infection more broadly, immunohistological analysis at autopsy of patients who deceased from SARS revealed that ACE2-expressing cells that were infected by SARS-CoV showed elevated expression of IL-6, IL-1&beta;, and TNF-&alpha; [@doi:10.1002/path.2067].
+Similarly, the introduction of the S-protein from SARS-CoV to mouse macrophages was found to increase production of IL-6 and TNF-&alpha; [@doi:10.1016/j.virusres.2007.02.007].
+For SARS-CoV-2 infection leading to COVID-19, early reports described a cytokine storm-like response in patients with particularly severe infections [@doi:10.1101/2020.02.24.963348; @doi:10/ggnzmc; @doi:10.1016/j.immuni.2020.06.017].
+Among patients hospitalized with COVID-19 in Wuhan, China, 59% developed sepsis, including 100% of the non-survivors [@doi:10.1016/S0140-6736(20)30566-3].
```

Comment:
Would it be helpful to have the number of total patients here to make more sense of the percentages?

Reply:
```suggestion
Among patients hospitalized with COVID-19 in Wuhan, China, 112 out of 191 (59%) developed sepsis, including all 54 of the non-survivors [@doi:10/ggnxb3].
```